test/e2e/play_kube_test.go

```diff
@@ -1085,6 +1104,20 @@ func getPersistentVolumeClaimVolume(vName string) *Volume {
 	}
 }
 
+// getConfigMap returns a new ConfigMap Volume given the name and items
+// of the ConfigMap.
+func getConfigMap(vName string, items []map[string]string, optional bool) *Volume {
```

Comment:
Is this function badly named. Your returning a volume not a configmap?

Reply:
Perhaps `getConfigMap` -> `getVolumeWithConfigMap`

Reply:
getVolumeConfigMap?
otherwise getCM and getConfigMap are very confusing.

Reply:
renamed to `getConfigMapVolume`